src/config/index.js

```diff
@@ -1,20 +1,22 @@
 import "regenerator-runtime/runtime";
 import AppSearchAPIConnector from "@elastic/search-ui-app-search-connector";
 
-function getConnector(env, lang) {
+function getConnector(env, lang, country) {
+  let countrySuffix = country ? `-${country}` : ''
+
   return new AppSearchAPIConnector({
-    engineName: "dokteronline-" + env + "-" + lang,
+    engineName: "dokteronline-" + env + "-" + lang + countrySuffix,
```

Comment:
Dit zou je nog mooi kunnen refactoren naar:

```suggestion
    engineName: `dokteronline-${env}-${lang}-${country ?? country}`,
```

Hierdoor kan je de `let` hierboven weghalen.